### DISCUSSION TITLE
Trouble with setting up the evolution

### DISCUSSION THREAD

**Discussion by AndrewHutani** [MilesCranmer/PySR#438] [Q&A]
Hi,

As a quick preface, I'm not very proficient with multiple topics brought up in the _readme_ and _Tuning and Workflow Tips_, such as julia and IPython. 
For context, I am using a conda environment and am mainly using Virtual Studio Code as a way of coding. As of now, the main part of the symbolic regression code looks like this:

```
model = PySRRegressor(
    niterations=1000,  # < Increase me for better results
    binary_operators=["+", "*", '-', '/'],
    unary_operators=[
        "cos",
        "sin",
    ],
    procs=8,
    populations=30,
    model_selection="best",
    maxsize = 15,

)

model.fit(X_train, y_train)
```
I managed to get IPython working, but it does not look like it is getting to the evolution part, since it does not show anything in the terminal. The last thing that is shown in the terminal is a warning regarding the static environment: 
```
UserWarning: Your system's Python library is static (e.g., conda), so precompilation will be turned off. For a dynamic library, try using `pyenv` and installing with `--enable-shared`: https://github.com/pyenv/pyenv/blob/master/plugins/python-build/README.md#building with---enable-shared. warnings.warn(
```

I was wondering what could be the problem regarding the evolution not starting (and maybe how to fix the static environment error, since I have not worked with pyenv)

If any extra information is neccesary, I would be happy to provide them
Thanks!

**Comment by MilesCranmer** [comment]
Could you share the size of your dataset? Also, when you say its not starting, can you see if it is running? (e.g., if the CPUs are in use).

**Reply by AndrewHutani** [reply]
The size of the dataset, both X_train and y_train, is (245, 6). Whenever I run the code, the system monitor does show that some of the CPUs are working. Is there any other way to check whether the code is still running?
